src/sdp/parser.rs

```diff
@@ -201,16 +202,22 @@ where
     ))
 }
 
-#[doc(hidden)]
-pub fn candidate<Input>() -> impl Parser<Input, Output = Candidate>
+/// Parse a candidate string into a [Candidate].
+///
+/// Does not parse an `a=` prefix.
+pub fn parse_candidate_attribute(s: &str) -> Result<Candidate, SdpError> {
```

Comment:
This name is a bit strange. The `a=` bit is literally the SDP attribute, so technically this function parses everything BUT the attribute.

I think we should turn it around so `candidate` and `candidate_attribute` change name with each other. That also drops the "attribute" bit from the module export which I think is correct.

Reply:
You beat me to it. I was going to reconsider these names but didn't get around to it yet.

Reply:
fn's renamed. Hopefully it makes sense now.